frontend/src/index.js

```diff
@@ -153,11 +153,32 @@ async function showFile(file, revision) {
     lines: source.split("\n").map((line, nb) => {
       const coverage = file.coverage[nb];
       let cssClass = "";
-      if (coverage && coverage !== -1) {
+      let tests = null;
```

Comment:
I'd call this "hits", as it is the number of times the line was executed during tests, not the number of tests which executed that line.

Reply:
Done